### DISCUSSION TITLE
PostgreSQL obtener el tamaño de tabla, índices y base de datos

### DISCUSSION THREAD

**Discussion by joseluisbossio** [OdooAR/odoo-argentina-doc#7] [Show and tell]
![image](https://user-images.githubusercontent.com/5479741/129174486-cffe6df1-de7d-4588-891b-36959b21cd82.png)

A veces necesitamos en PostgreSQL obtener el tamaño de tabla, índices y base de datos 

Dejo un enlace que me pareció interesante [How to Get Table, Database, Indexes, Tablespace, and Value Size in PostgreSQL](https://www.postgresqltutorial.com/postgresql-database-indexes-table-size/)

¿Conocen otras formas?

**Comment by ibuioli** [comment]
Para ver tablas yo siempre utilicé el comando `\dt` y `\dt+`
Acá hay más info: https://www.postgresqltutorial.com/postgresql-show-tables/
